main.go

```diff
@@ -410,3 +427,32 @@ func tlsConfigSetting(cfg *tls.Config, tlsOpt tlsConfig) {
 	}
 	cfg.CipherSuites = cipherSuiteIDs
 }
+
+func checkRbacPermissions(reviewer *rbac.Reviewer, ctx context.Context) (admission.Warnings, error) {
```

Comment:
Instead of keeping this method here, could we move it to the autodetect package and follow the pattern we have for environment detection? That would allow us to keep with our current style for these checks:
```
if cfg.RBACPermissions() == permissions.Allowed {
	setupLog.Info("Has necessary RBAC permissions.")
	...
} else {
	setupLog.Info("Does not have necessary RBAC permissions, skipping.")
}
```